analysers/analyser_merge_charging_station_FR.py

```diff
@@ -73,21 +212,142 @@ def __init__(self, config, logger=None):
                     },
                     mapping2={
                         "email": "contact_operateur",
-                        "phone": "telephone_operateur",
+                        "operator:phone": "telephone_operateur",
+                        "operator:email": "contact_operateur",
+                        "start_date": "date_mise_en_service",
                         "capacity": "nbre_pdc",
+                        "charging_station:output": lambda fields: self.socket_output_find_correspondances(
+                            fields["puissance_nominale"]) if fields["puissance_nominale"] else None,
                         "bicycle": lambda fields: "yes" if fields["station_deux_roues"] == "true" else None,
                         "motorcycle": lambda fields: "yes" if fields["station_deux_roues"] == "true" else None,
                         "moped": lambda fields: "yes" if fields["station_deux_roues"] == "true" else None,
                         "motorcar": lambda fields: "no" if fields["station_deux_roues"] == "true" else "yes",
                         "opening_hours": "horaires_grouped",
-                        "fee": lambda fields: "yes" if fields["gratuit_grouped"] == "false" else ("no" if fields["gratuit_grouped"] == "true" else None),
-                        "authentication:none": lambda fields: "yes" if fields["paiement_acte_grouped"] == "true" else None,
-                        "payment:credit_cards": lambda fields: "yes" if fields["paiement_cb_grouped"] == "true" else ("no" if fields["paiement_cb_grouped"] == "false" else None),
+                        "fee": lambda fields: "yes" if fields["gratuit_grouped"] == "false" else (
+                            "no" if fields["gratuit_grouped"] == "true" else None),
+                        "authentication:none": lambda fields: "yes" if fields[
```

Comment:
Not sure what these changes are good for, looks like a strange place for a line break.
(There's more of these)

Reply:
that what was given by auto formatting, i changed it to single line